Makefile

```diff
@@ -211,6 +211,7 @@ dev-install-node-modules:
 	sed -ie 's/$(NNI_VERSION_TEMPLATE)/$(NNI_VERSION_VALUE)/' $(NNI_PKG_FOLDER)/package.json
 	ln -sf ${PWD}/src/nni_manager/node_modules $(NNI_PKG_FOLDER)/node_modules
 	ln -sf ${PWD}/src/webui/build $(NNI_PKG_FOLDER)/static
+	mkdir -p $(NASUI_PKG_FOLDER)
 	ln -sf ${PWD}/src/nasui/build $(NASUI_PKG_FOLDER)/build
 	ln -sf ${PWD}/src/nasui/server.js $(NASUI_PKG_FOLDER)/server.js
```

Comment:
needs to remove this line, or will give 
```
# Installing NNI Package 
rm -rf /home/yann/.local/nni
ln -sf /home/yann/code/nni/src/nni_manager/dist /home/yann/.local/nni
cp src/nni_manager/package.json /home/yann/.local/nni
sed -ie 's/999.0.0-developing/v0.2-1270-g06bc964/' /home/yann/.local/nni/package.json
ln -sf /home/yann/code/nni/src/nni_manager/node_modules /home/yann/.local/nni/node_modules
ln -sf /home/yann/code/nni/src/webui/build /home/yann/.local/nni/static
mkdir -p /home/yann/.local/nni/nasui
ln -sf /home/yann/code/nni/src/nasui/build /home/yann/.local/nni/nasui/build
ln -sf /home/yann/code/nni/src/nasui/server.js /home/yann/.local/nni/nasui/server.js
ln: '/home/yann/code/nni/src/nasui/server.js' and '/home/yann/.local/nni/nasui/server.js' are the same file
Makefile:207: recipe for target 'dev-install-node-modules' failed
make: *** [dev-install-node-modules] Error 1
```

Reply:
Didn't hit this issue, and it's not changed this time.